### ISSUE TITLE
LSST @ Vera Rubin Observatory

### ISSUE THREAD

**Issue by bersavosh** [ethrane/transients#13]
LSST at Rubin observatory will provide a unique factory of transient discovery and follow-up in the optical band. The survey is supposed to being in the next year or two. There is currently [Australian investment and involvement](https://astronomyaustralia.org.au/blog/portfolio/vera-c-rubin-observatory/) and multiple groups across Australia are working on various aspects of the survey in preparation.

Given the broad science cases this facility covers, existing involvement, and exciting prospects, it may be a infrastructure priority  for our working group to indicate the importance of Australian involvement and maintaining data rights.

**Comment by anaismoller** [comment]
LSST is expected to start mid 2025. 

One important factor is that Australia is the only place to observe transients detected by Rubin in Chile minutes to hours after as Earth turns. So it can be mentioned that this priority also adds value to existing Australian facilities and surveys.

**Comment by anaismoller** [comment]
There will be also, depending on schedules, an overlap with other priorities such as SKA.

**Comment by jeffcooke** [comment]
This is what is covered by the previous Infrastructure Priority question "Access to Australian facilities and instrumentation to meet the needs for transient discovery and follow up". Could they be merged?

**Comment by bersavosh** [comment]
Hi Jeff, I thought the emphasis in #10 was the Australian nature/location of facilities and instruments - perhaps some rewording might enable encompassing this under that?

**Comment by jeffcooke** [comment]
Yes, some rewording of #10 could include #13.  #10 is an infrastructure need and the was to address the unique position we are in (figuratively and literally) for transient follow up of Rubin and to encompass all that's needed for that. I included a number of things I could think of including Rubin nightly data (which is made public), the Fink data broker, and our need for facilities to respond and follow up. But we would certainly need data rights for the survey data, too.
